pkg/providers/cloudstack/config/template-cp.yaml

```diff
@@ -173,6 +173,55 @@ spec:
 {{ .schedulerExtraArgs.ToYaml | indent 10 }}
 {{- end }}
     files:
+{{- if .cloudstackKubeVip}}
+    - content: |
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          creationTimestamp: null
+          name: kube-vip
+          namespace: kube-system
+        spec:
+          containers:
+          - args:
+            - start
+            env:
+            - name: vip_arp
+              value: "true"
+            - name: vip_leaderelection
+              value: "true"
+            - name: vip_address
+              value: {{.controlPlaneEndpointHost}}
+            - name: vip_interface
+              value: ens3
+            - name: vip_leaseduration
+              value: "15"
+            - name: vip_renewdeadline
+              value: "10"
+            - name: vip_retryperiod
+              value: "2"
+            image: {{.kubeVipImage}}
+            imagePullPolicy: IfNotPresent
+            name: kube-vip
+            resources: {}
+            securityContext:
+              capabilities:
+                add:
+                - NET_ADMIN
+                - SYS_TIME
```

Comment:
What's the `SYS_TIME` capability required for?

Reply:
I'm not able to find much documentation about this attribute [besides cryptic source-code notes](https://github.com/torvalds/linux/blob/master/include/uapi/linux/capability.h) but it seems present in a lot of examples using kube-vip.